src/main/java/racingcar/entry/CarPosition.java

```diff
@@ -12,8 +12,8 @@ public CarPosition(int position) {
         this.position = position;
     }
 
-    public void increase() {
-        position++;
+    public int increase() {
+        return position + 1;
     }
```

Comment:
```suggestion
    public Position increase() {
        return new Position(position + 1);
    }
```

불변이니까 증가된 값을 줘도 될 것 같아요

Reply:
넹~😍